foundations/html_css/flexbox/flexbox-axes.md

```diff
@@ -22,42 +23,44 @@ The default direction for a flex container is horizontal, or `row`, <span id='fl
 
 The example above fixed this by specifying `flex: 1 1 auto`, telling the flex items to default to their given `height`. We could also have fixed it by putting a height on the parent `.flex-container`, or by using `flex-grow: 1` instead of the shorthand.
 
 Another detail to notice: when we changed the <span id='column-flex-basis'>flex-direction to `column`, `flex-basis` refers to `height` instead of `width`.</span> Given the context this may be obvious, but it's something to be aware of.
 
-We've strayed from the point slightly... We were talking about flex-direction and axes. To bring it back home, the default behavior is `flex-direction: row` which arranges things horizontally. The reason this often works well without changing other details in the CSS is because block-level elements default to the full width of their parent. Changing things to vertical using `flex-direction: column` adds complexity because block-level elements default to the height of their content, and in this case there _is_ no content.
+We've strayed from the point slightly... We were talking about flex-direction and axes. To bring it back home, the default behavior is `flex-direction: row` which arranges things horizontally. The reason this often works well without changing other details in the CSS is because block-level elements default to the full width of their parent. Changing things to vertical using `flex-direction: column` adds complexity because block-level elements default to the height of their content, and in this case there *is* no content.
 
 > There are situations where the behavior of flex-direction could change if you are using a language that is written top-to-bottom or right-to-left, but you should save worrying about that until you are ready to start making a website in Arabic or Hebrew.
 
-
 ### Knowledge check
 
 This section contains questions for you to check your understanding of this lesson on your own. If you’re having trouble answering a question, click it and review the material it links to.
 
--   [How do you make flex items arrange themselves vertically instead of horizontally?](#flex-vertical)
--   [In a `column` flex-container, what does `flex-basis` refer to?](#column-flex-basis)
--   [In a `row` flex-container, what does `flex-basis` refer to?](#row-flex-basis)
--   [Why do the previous two questions have different answers?](#flex-axes)
+- [How do you make flex items arrange themselves vertically instead of horizontally?](#flex-vertical)
+- [In a `column` flex-container, what does `flex-basis` refer to?](#column-flex-basis)
+- [In a `row` flex-container, what does `flex-basis` refer to?](#row-flex-basis)
+- [Why do the previous two questions have different answers?](#flex-axes)
 
 ### Additional resources
 
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
```

Comment:
@MaoShizhong, the concern I raised in [line 53](https://github.com/TheOdinProject/curriculum/pull/27855#discussion_r1580309477) applies to here as well.

Reply:
That one's real hard to see, but for some reason the apostrophe in `isn’t` is a weird alternative apostrophe character.

Instead of `isn’t`, it should be  `isn't`